### DISCUSSION TITLE
[Projects Beta] - Filter Iteration by Current flag

### DISCUSSION THREAD

**Discussion by iBaldie** [community/community#6478] [Projects and Issues] [answered]
There is a Current, Planned and No Iteration status, I want to add this to a filter to create a View for my Current Iteration.

Instead of just filtering by an iteration Name.

**Comment by daanleenders** [comment]
Would like that too. And if it is possible a way to filter completed Iterations. 

Strangely completed iterations are also still marked as planned. They go to a 'completed' tab in the settings but show up as 'planned' in that tab. I wouldn't expect issues in a completed iteration to show up when you filter for planned iterations.

**Comment by zaporylie** [comment, marked as answer]
It is now possible to filter by current sprint by using the "@current" value for the iteration field, ex. sprint:"@current"
